### ISSUE TITLE
Impossible cell voltage

### ISSUE THREAD

**Issue by najaB76** [slim-bean/powermon#2]
Hello @slim-bean ,

I've been using dbus-serialbattery which has forked from your code.  First off, great job decoding the protocol, it's been really useful to get accurate battery state readings, better than a smart shunt!

One thing that I have noticed is that when battery gets close to full the individual cell voltages will return values that don't make sense.  I've seen it reading 36V for the individual cells.  My guess is that this is when balancing is active.  In dbus-spy the values look like this:
```
Voltages/Cell1 3.512
Voltages/Cell2 36.276
Voltages/Cell3 36.281
Voltages/Cell4 36.277
Voltages/Cell5 3.493
Voltages/Cell6 3.509
Voltages/Cell7 3.513
Voltages/Cell8 36.278

```
I switched the RS485 across to my PC and read the values, so this was maybe a minute or so after the dbus-spy readings above:

![image](https://user-images.githubusercontent.com/96979341/220586796-acaf6080-e7bc-4b15-9a3a-6d8172822fe3.png)

I was thinking a first that it might simply be a case of subtracting 32 from the value but that still gives a cell voltage of 4.2 volts which is still high for a cell voltage.  Any ideas what's going on?

Thanks

**Comment by pchiquit** [comment]
Hi @najaB76 
   I fixed that bug on the latest version of the EG4 Lifepower driver,  but I don't think it was released yet. https://github.com/Louisvdw/dbus-serialbattery/pull/390 
   I would suggest we move the discussion to [Louisvdw](https://github.com/Louisvdw)/[dbus-serialbattery](https://github.com/Louisvdw/dbus-serialbattery) as @slim-bean is not involved on the dbus-serialbattery development. 

Basically I'm masking that bit that throws the voltage off, but your debugging gave me the idea of also reporting the cells that are balancing to the serial driver. I'll see if that is possible.

**Comment by slim-bean** [comment]
I have seen this spike in voltage myself a few times, good to know it's a masking issue! I'll see about making that change here as well
